src/app/modules/main/communities/module.nim

```diff
@@ -252,6 +252,12 @@ method communityMuted*(self: Module, communityId: string, muted: bool) =
 method communityAccessRequested*(self: Module, communityId: string) =
   self.view.communityAccessRequested(communityId)
 
+method communityHistoryArchivesDownloadStarted*(self: Module, communityId: string) =
+  self.view.setDownloadingCommunityHistoryArchives(true)
+
+method communityHistoryArchivesDownloadFinished*(self: Module, communityId: string) =
+  self.view.setDownloadingCommunityHistoryArchives(false)
```

Comment:
We're not using `communityId` here **yet** until we've decided how we want to handle visual indication of ongoing history archive downloads on a per community basis